packages/shared/src/graphql/feedSettings.ts

```diff
@@ -1,17 +1,23 @@
 import { gql } from 'graphql-request';
 import { Source } from './sources';
 
+export enum AdvancedSettingsGroup {
+  Advanced = 'advanced',
+  ContentTypes = 'content_types',
+}
+
 export interface AdvancedSettings {
   id: number;
   title: string;
   description: string;
   defaultEnabledState: boolean;
+  group: AdvancedSettingsGroup;
 }
 
-export interface FeedAdvancedSettings
-  extends Partial<Omit<AdvancedSettings, 'defaultEnabledState'>> {
```

Comment:
We're really not using the extended properties, so no point having it here, but instead, it should be from the `advancedSettings` property.